bikeshed/__init__.py

```diff
@@ -2551,8 +2558,15 @@ def inlineRemoteIssues(doc):
                 if el.tag == "p":
                     el.tag = "div"
                 addClass(el, "no-marker")
-        except:
-            pass
+        except urllib2.HTTPError as err:
+            if doc.token and err.code == 401:
+                die("Unauthorized Access to GitHub's API. There might be an issue with your token.")
```

Comment:
Please put a `continue` or `break` after each of these die()s, as appropriate.  There's nothing after this block *now*, but there might be in the future, so it's best to make sure the die() properly stops working on the iteration as soon as it find an error.